libbeat/template/processor.go

```diff
@@ -442,14 +453,14 @@ func (p *Processor) object(f *mapping.Field) common.MapStr {
 }
 
 type dynamicTemplateKey struct {
-	path      string
+	name      string
 	pathMatch string
 	matchType string
 }
 
-func (p *Processor) addDynamicTemplate(path string, pathMatch string, properties common.MapStr, matchType string) {
+func (p *Processor) addDynamicTemplate(name, pathMatch, matchType string, properties common.MapStr) bool {
```

Comment:
It seems you now return a `bool`, but this is never used, right?

Reply:
Indeed. I meant to revert that; done now. Thanks!